### PR TITLE
Remove warnings

### PR DESCRIPTION
**Pull request by ThatTobMate**
Added in the prop-types and create-react-class packages.
Also updated enzyme to the latest version which has those packages too.

Let me know if you think i've missed anything

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "qubit-react",
-  "version": "1.1.2",
+  "version": "1.1.3",
```
Comment (KwanMan) [inline package.json]: Will be a minor release, it's not really a bug fix, it's adding support for extra versions. But we do version bumps in separate commits anyway when we release so can remove this change from the PR

Comment (ThatTobMate) [inline package.json]: ok cool
